lectures/commod_price.md

```diff
@@ -134,8 +133,7 @@ $p_t$.
 
 The harvest of the commodity at time $t$ is $Z_t$.
 
-We assume that the sequence $\{ Z_t \}_{t \geq 1}$ is IID with common
-density function $\phi$.
+We assume that the sequence $\{ Z_t \}_{t \geq 1}$ is {ref}`IID <lln_ksl>` with common density function $\phi$, where $\phi$ is nonnegative.
```

Comment:
@SylviaZhaooo is this point directly to the definition of IID or a lecture on law of large numbers?

Reply:
Thank you so much for pointing that out. I have updated the link.